BitbucketServerAutomation/Functions/Find-BBServerRepository.ps1

```diff
@@ -0,0 +1,17 @@
+function Find-BBServerRepository
+{
+    param(
+        [parameter(Mandatory=$true)]
+        [Object]
+        $Connection,
+        # Name of a Repository you wish to find. supports Wildcards. 
+        [String]
+        $Name
+    )
+    $repositoryList = @()

```

Comment:
Remove `$repositoryList`. Let the pipeline return any repository object that matches the `Where-Object` condition.